server/channels/store/storetest/post_store.go

```diff
@@ -776,7 +776,7 @@ func testPostStoreGetForThread(t *testing.T, rctx request.CTX, ss store.Store) {
 		require.NoError(t, err)
 		require.Len(t, r1.Order, 3) // including the root post
 		require.Len(t, r1.Posts, 3)
-		assert.True(t, r1.HasNext)
+		require.Nil(t, r1.HasNext)
```

Comment:
These changes seem wrong. The assertion should still be true here. I would assume something like: `assert.True(t, *r1.HasNext)` after a `require.NotNil(t, r1.HasNext)`